packages/kbn-optimizer/limits.yml

```diff
@@ -105,7 +105,7 @@ pageLoadAssetSize:
   screenshotting: 22870
   searchprofiler: 67080
   security: 65433
-  securitySolution: 273763
+  securitySolution: 339077
```

Comment:
Can resolving https://github.com/elastic/kibana/issues/95870 please be prioritized? It's affecting the end-user performance of Kibana as a whole and just appears to be getting worse, not better.

Reply:
That will come with this [PR](https://github.com/elastic/kibana/pull/141010)

Reply:
Awesome, thanks!

Reply:
I think after updating this PR with the main, which contains the recent bundle changes, we can set securitySolution: 66738 back.

Reply:
@YulNaumenko @tylersmalley I managed to keep 66738 in this PR. Will see if I can reduce it a bit more by doing https://github.com/elastic/kibana/issues/143532